src/NLog/Layouts/SimpleLayout.cs

```diff
@@ -138,6 +141,11 @@ public string Text
         /// </summary>
         public string FixedText => _fixedText;
 
+        /// <summary>
+        /// Is the message a simple formatted string? (Can skip StringBuilder)
+        /// </summary>
+        internal bool IsSimpleString => _stringValueRenderer != null;
```

Comment:
Was thinking of `IsSingleStringText`. "simple" is a bit ambiguous here.

Reply:
I think the name of `_stringValueRenderer ` should be in-sync with "simpleString" (or the new name)

Reply:
IsSingleStringText is fine with me. Will fix this